lib/bitcoinerlab.js

```diff
@@ -0,0 +1,144 @@
+const secp256k1 = require('@exodus/bitcoinerlab-secp256k1')
+let elliptic
+
+const verbose = false
+function loadLegacy (method, message) {
+  if (verbose) {
+    if (!message) {
+      console.warn(`${method} is not implemented, using legacy approach...`)
+    } else console.warn(`${method} | ${message}`)
+  }
+  if (!elliptic) {
+    elliptic = require('./elliptic')
+  }
+}
+
+function isCompressedPublicKey (array) {
+  return array.length === 33
+}
+
+module.exports = {
+  contextRandomize: function () {
+    return 0
+  },
+  privateKeyVerify: function (privateKey) {
+    return secp256k1.isPrivate(privateKey) ? 0 : 1
+  },
+  privateKeyNegate: function (privateKey) {
+    try {
+      privateKey.set(secp256k1.privateNegate(privateKey))
+    } catch (e) {
+      loadLegacy('privateKeyNegate', e.toString())
+      elliptic.privateKeyNegate(privateKey)
+    }
+    return 0
+  },
+  privateKeyTweakAdd: function (privateKey, tweak) {
+    try {
+      privateKey.set(secp256k1.privateAdd(privateKey, tweak))
+    } catch (e) {
+      return 1
+    }
+    return 0
+  },
+  privateKeyTweakMul: function (privateKey, tweak) {
+    loadLegacy('privateKeyTweakMul')
+    return elliptic.privateKeyTweakMul(privateKey, tweak)
+  },
+  publicKeyVerify: function (publicKey) {
+    return secp256k1.isPoint(publicKey) ? 0 : 1
+  },
+  publicKeyCreate: function (output, seckey) {
+    try {
+      output.set(secp256k1.pointFromScalar(seckey, isCompressedPublicKey(output)))
```

Comment:
`secp256k1.pointFromScalar(...)`  can return null, please throw on that rather than setting `output` to null.